[2023-04-06T10:50:13.059+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T10:40:00+00:00 [queued]>
[2023-04-06T10:50:13.074+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T10:40:00+00:00 [queued]>
[2023-04-06T10:50:13.077+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T10:50:13.079+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-06T10:50:13.080+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T10:50:13.102+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 10:40:00+00:00
[2023-04-06T10:50:13.114+0000] {standard_task_runner.py:55} INFO - Started process 12117 to run task
[2023-04-06T10:50:13.122+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T10:40:00+00:00', '--job-id', '222', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmpbsx7onrp']
[2023-04-06T10:50:13.127+0000] {standard_task_runner.py:83} INFO - Job 222: Subtask respond
[2023-04-06T10:50:13.242+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T10:40:00+00:00 [running]> on host cfece00ff772
[2023-04-06T10:50:13.380+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T10:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T10:40:00+00:00
[2023-04-06T10:50:13.384+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-06T10:50:13.420+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T104000, start_date=20230406T105013, end_date=20230406T105013
[2023-04-06T10:50:13.455+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T10:50:13.488+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
